client/blocks/get-apps/desktop-download-card.jsx

```diff
@@ -267,13 +299,33 @@ class DesktopDownloadCard extends Component {
 
 		return (
 			<>
-				<Button
-					className="get-apps__desktop-button"
-					href={ localizeUrl( this.getButtonLink( platform ) ) }
-					onClick={ this.getButtonClickHandler( platform ) }
-				>
-					{ this.getButtonText( platform ) }
-				</Button>
+				{ platform === 'MacIntel' ? (
+					<>
+						<Button
+							className="get-apps__desktop-button"
+							href={ localizeUrl( this.getButtonLink( platform )[ 0 ] ) }
+							onClick={ this.getButtonClickHandler( platform ) }
+						>
+							{ this.getButtonText( platform )[ 0 ] }
+						</Button>
+						<Button
+							className="get-apps__desktop-button"
+							href={ localizeUrl( this.getButtonLink( platform )[ 1 ] ) }
+							onClick={ this.getButtonClickHandler( platform ) }
+						>
+							{ this.getButtonText( platform )[ 1 ] }
+						</Button>
+					</>
```

Comment:
Having a separate case for the Mac Silicon, we would be more explicit about what we want to show.

```suggestion
				{ platform === 'MacIntel' ? (
					<>
						<Button
							className="get-apps__desktop-button"
							href={ localizeUrl( this.getButtonLink( platform ) ) }
							onClick={ this.getButtonClickHandler( platform ) }
						>
							{ this.getButtonText( platform ) }
						</Button>
						<Button
							className="get-apps__desktop-button"
							href={ localizeUrl( this.getButtonLink( PLATFORM_MAC_SILICON ) ) }
							onClick={ this.getButtonClickHandler( PLATFORM_MAC_SILICON ) }
						>
							{ this.getButtonText( PLATFORM_MAC_SILICON ) }
						</Button>
					</>
```